packages/react-scripts/config/webpack.config.ssr.js

```diff
@@ -82,7 +88,7 @@ module.exports = function(webpackEnv) {
     : isEnvDevelopment && '/';
   // Some apps do not use client-side routing with pushState.
   // For these, "homepage" can be set to "." to enable relative asset paths.
-  const shouldUseRelativeAssetPaths = publicPath === './';
+  // const shouldUseRelativeAssetPaths = publicPath === './';
```

Comment:
Could you confirm why this has been removed?

Reply:
It was only used by `MiniCssExtractPlugin`